src/main/common/clients/launchDarklyClient.ts

```diff
@@ -0,0 +1,25 @@
+import * as config from 'config'
+import { User } from 'idam/user'
+import * as ld from 'ldclient-node'
+
+const sdkKeys: string = config.get<string>('launchDarkly.sdk-keys.production')
+
+export class LaunchDarklyClient {
+  static client: ld.LDClient
+  static initClient () {
```

Comment:
think about whether an instance using a constructor would make more sense, not sure

Reply:
I have now implemented this as a Singleton as per recommendation from LaunchDarkly documentation (https://docs.launchdarkly.com/docs/node-sdk-reference)